lib/send_text.rb

```diff
@@ -0,0 +1,21 @@
+require 'twilio-ruby'
+
+module SendText
+  def send_text(basket_content, total)
+    account_sid = 'ACf7689020829b6d37eb7efb1262ab9f40'
+    auth_token = 'c1877557e592e48ae2456ee1d072f89e'
+
+    @client = Twilio::REST::Client.new account_sid, auth_token
+
+    delivery_time = (Time.now + 60*60).strftime("%H:%m")
+
+    text =  "Your order has been confirmed!." +
```

Comment:
Style/LineEndConcatenation: Use `\` instead of `+` or `<<` to concatenate those strings.